src/Components/ImagesTable/ImagesTableToolbar.tsx

```diff
@@ -141,6 +150,20 @@ const ImagesTableToolbar: React.FC<imagesTableToolbarProps> = ({
             ouiaId="blueprint-out-of-sync-alert"
           />
         )}
+        {blueprintsComposes &&
+          blueprintsComposes.data.length > 0 &&
+          isBlueprintDistroCentos8() && (
+            <Alert
+              style={{
+                margin:
+                  '0 var(--pf-v5-c-toolbar__content--PaddingRight) 0 var(--pf-v5-c-toolbar__content--PaddingLeft)',
+              }}
+              isInline
+              variant="warning"
+              title="CentOS Stream 8 is end of life, building images from this blueprint will fail."
```

Comment:
What about? ```suggestion
              title="CentOS Stream 8 is no longer supported, building images from this blueprint will fail."
```

I think I live the wording more, wdyt? We can also link to the announcement, but if that's too hard, it's not needed: https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/

Reply:
Updated the wording :+1: 

Not sure about the link. I think the only links we currently use lead to documentation or other services. :thinking: I'm not against adding it, but I'm not sure if it's okay to link "outside".